codec-dns/src/main/java/io/netty/handler/codec/dns/DnsCodecUtil.java

```diff
@@ -120,7 +120,7 @@ static String decodeDomainName(ByteBuf in) {
 
     /**
      * Decompress pointer data.
-     * @param compression comporession data
+     * @param compression compression data
```

Comment:
We should change it to `compressed` data.

Reply:
done